.devcontainer/Dockerfile

```
@@ -0,0 +1,4 @@
+FROM nixos/nix
```

Hadolint (2.12.0)

[warning] Always tag the version of an image explicitly

(DL3006)